doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -43,34 +43,317 @@ On your computer, one of the following operating systems:
 
 |supported OS|
 
-You also need to install `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively).
+You also need the following:
 
-.. _ug_nrf54h20_gs_test_blinky:
+* `Git`_ or `Git for Windows`_ (on Linux and Mac, or Windows, respectively)
+* `curl`_
+* SEGGER `J-Link version 7.94e`_
 
-Testing with the Blinky sample
-******************************
+.. _ug_nrf54h20_gs_installing_software:
 
-The nRF54H20 DK comes preprogrammed with the :zephyr:code-sample:`blinky` sample.
+Installing the required software
+********************************
 
-Complete the following steps to test that the DK works correctly:
+To work with the nRF54H20 DK, follow the instructions in the next sections to install the required tools.
 
-1. Connect the USB-C end of the USB-C cable to the **IMCU USB** port the nRF54H20 DK.
-#. Connect the other end of the USB-C cable to your PC.
-#. Move the **POWER** switch to **On** to turn the nRF54H20 DK on.
-   The switch is located in the top left corner of the DK PCB.
+.. _ug_nrf54h20_install_toolchain:
 
-**LED1** will turn on and start to blink.
+Installing the toolchain
+========================
 
-If something does not work as expected, contact Nordic Semiconductor support.
+You can install the toolchain for the customer sampling of the |NCS| by running an installation script.
 
-.. _nrf54h20_gs_installing_software:
+.. tabs::
 
-Installing the required software
-********************************
+   .. tab:: Windows
+
+      Follow these steps:
+
+      1. Open Git Bash.
+      #. Download and run the :file:`bootstrap-toolchain.sh` installation script file using the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
+
+         Depending on your connection, this might take some time.
+      #. Open a new terminal window with the |NCS| toolchain environment by running the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+            c:/ncs-lcs/nrfutil.exe toolchain-manager launch --terminal --chdir "c:/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+
+         This setup allows you to access west and other development tools.
+         Alternatively, you can set up the environment variables manually by running the following command::
+
+            c:/ncs-lcs/nrfutil.exe toolchain-manager env --as-script
+
+         Copy-paste the output into the terminal and execute it to enable the use of west directly in that window.
+
+         .. note::
+            When working with the customer sampling release, you must always use a terminal window where the west environmental variables have been called.
```

Comment:
This still does not match Lines 121 and 177.

Reply:
It cannot match, it describes a different scenario. On Linux and MacOS we are instructing just how to open a terminal window having toolchain ready to use.
On Windows we are describing 2 different options, and they can choose any of these.

Reply:
I would still expect their wording to be closer to the wording for the Windows version, something along the lines of:
```
When working with west in the customer sampling release, you must always use a shell window with the |NCS| toolchain environment.
```